style.css

```diff
@@ -0,0 +1,269 @@
+.nav-btns-left {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	gap: 15px;
+	margin-left: 20px;
+	margin-top:10px;
+	color: #000000;
+}
+
+.nav-btn-right {
+	
+	display: flex;
+	justify-content: flex-end;
+	margin-top: 10px; 
+	margin-right: 15px;
+}
+
+.nav-btn-right a , .nav-btns-left a {
+	text-decoration: none;
+	font-size: 1.5rem;
+	
+}
+
+.link {
+	color: #f7f7f7;
+}
+.link:hover {
+	color: #a0a5ad;
+}
+/*IMGS MAIN SECTION*/
+.imgs-container {
+	display: flex;
+	flex-wrap: wrap;
+	width: 50%;
+	margin: 5px 10px;
+}
+.imgs-container img {
+	width: 50%;
+	aspect-ratio: 4/3;
+	object-fit: cover;
+	
+	padding: 5px;
+}
+/*HELLO SECTION*/
+.hello-section {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	justify-content: center;
+	width: 50%;
+	padding: 15px;
+}
+.hello-section h1 {
+	font-size: 4rem;
+	align-items: center;
+	color:#f7f7f7;
+}
+.hello-section h2 {
+	font-size: 3rem;
+	color:#f7f7f7;
+}
+/*ABOUT SECTION*/
+.about-section {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	padding: 0 10px;
+}
+.about-section h2 {
+	font-size: 2rem;
+	margin-left: 15px;
+	color:#f7f7f7;
+}
+.about-section h3 {
+	font-size: 1.5rem;
+	padding: 0;
+	margin-top: 0;
+	margin-left: 15px;
+	color: #f7f7f7;
+	width:100%;
+	
+}
+/*PORTFOLIO SECTION*/
+.portfolio-section {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: column;
+	gap: 10px;
+}
+.portfolio-description {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	background-color: #000000;
+	padding: 0 10px;
+}
+.portfolio-description h2 {
+	font-size: 2rem;
+	margin-left: 15px;
+	color: #D9D9D9;
+}
+.portfolio-description h3 {
+	font-size: 1.5rem;
+	padding: 0;
+	margin-top: 0;
+	margin-left: 15px;
+	color: #D9D9D9;
+}
+.card {
+	display: flex;
+	flex-wrap: wrap;
+	width: 100%
+	flex-direction: column;
+	max-width: 1200px;
+	margin-left: auto;
+	margin-right: auto;
+	border: 1px solid black;
+	border-radius: 20px;
+	
+}
+.card h3 {
+	font-size: 2rem;
+	margin-left: 15px;
+	margin-bottom: 10px;
+	padding: 0 30px;
+	margin-left: 10px;
+	flex-direction: row;
+	padding-top: 0;
+	margin-top: 15px;
+	color:#f7f7f7;
+	
+}
+.card-img-text {
+	display: flex;
+	flex-wrap: nowrap;
+	margin-left: auto;
+	margin-right: auto;
+	flex-direction: row;
+}
+.card-img-text img {
+	width: 400px;
+	aspect-ratio: auto;
+	justify-content: flex-start;
+	margin-left: 10px;
+	margin-bottom: 10px;
+	border-radius: 10px;
+	border: 1px solid #d9d9d9;
+	
+}
+.card p {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	padding: 0 40px;
+	font-size: 1.5rem;
+	color:#f7f7f7;
+}
+
+/*CONTACT SECTION*/
+.contact-section {
+	margin-top: 10px;
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	justify-content: center;
+}
+
+.contact-section h2 {
+	font-size: 1.75rem;
+	margin-left: 15px;
+	margin-bottom:0;
+	color:#f7f7f7;
+}
+.contact-section h3 {
+	font-size: 1.5rem;
+	margin-left: 15px;
```

Comment:
when we put left margin, we usually also put right margin, unless there is a big reason not to